packages/framework-types/src/concepts/read-model.ts

```diff
@@ -14,4 +16,14 @@ export interface ReadModelMetadata {
   readonly class: Class<ReadModelInterface>
   readonly properties: Array<PropertyMetadata>
   readonly authorizedRoles: RoleAccess['authorize']
+  readonly before: NonNullable<FilterHooks['before']>
 }
+
+export interface FilterHooks {
+  readonly before?: Array<BeforeFunction>
+}
```

Comment:
# This is the part that should be modified accordingly to my explanation above

Reply:
😂 😂 😂 😂 😂  It seems you are shouting! 

Anyway, check my response to that comment 😉